openfeature/util_test.go

```diff
@@ -62,3 +62,18 @@ func (s ProviderEventing) Invoke(e Event) {
 func (s ProviderEventing) EventChannel() <-chan Event {
 	return s.c
 }
+
+func eventually(t *testing.T, condition func() bool, timeout, interval time.Duration, errMsg string) {
```

Comment:
I saw you using this and I thought it was some library I didn't know about. We've built the same helper in other repos as well. :joy:

Reply:
😄 yeah, actually i am used to [testify](https://github.com/stretchr/testify) which provides such functionalities.
This repository is not using any external testing library (which is also fine) so I just "borrowed" the idea and naming from there 😛

Reply:
I don't mind adding testify at all, since it wouldn't be shipped. If you feel it's a good addition, go for it.

Reply:
Got it. To keep the test assertions consistent, I wouldn't bring in a new third-party library just for this tho - unless there's a really compelling reason.